$NetBSD: patch-make-include-main.defs,v 1.5 2023/04/09 11:15:50 plunky Exp $

Use comon pkgsrc libraries, except for libjpeg-turbo which is required but
cannot coexist with libjpeg (required by gtk3)

--- make/include/main.defs.orig	2023-01-22 16:36:49.000000000 +0000
+++ make/include/main.defs
@@ -13,49 +13,12 @@ ifneq ($(HAS.bz2),1)
 endif
 endif
 
-ifneq (,$(filter $(HOST.system),darwin cygwin mingw))
-    MODULES += contrib/freetype
-    MODULES += contrib/fribidi
-    MODULES += contrib/harfbuzz
-    MODULES += contrib/libxml2
-    MODULES += contrib/libass
-    MODULES += contrib/libogg
-    MODULES += contrib/libvorbis
-    MODULES += contrib/libopus
-    MODULES += contrib/libspeex
-    MODULES += contrib/libtheora
-    MODULES += contrib/lame
-    MODULES += contrib/x264
-    MODULES += contrib/jansson
-    MODULES += contrib/libvpx
-    MODULES += contrib/libjpeg-turbo
-endif
-
-ifeq (1,$(FEATURE.flatpak))
-    MODULES += contrib/libass
-    MODULES += contrib/jansson
-    MODULES += contrib/x264
-endif
+MODULES += contrib/libjpeg-turbo
 
 ifeq (1,$(FEATURE.fdk_aac))
     MODULES += contrib/fdk-aac
 endif
 
-ifeq (1,$(FEATURE.x265))
-    MODULES += contrib/x265
-    MODULES += contrib/x265_8bit
-    MODULES += contrib/x265_10bit
-    MODULES += contrib/x265_12bit
-endif
-
-MODULES += contrib/libdav1d
-MODULES += contrib/svt-av1
-MODULES += contrib/zimg
-MODULES += contrib/ffmpeg
-MODULES += contrib/libdvdread
-MODULES += contrib/libdvdnav
-MODULES += contrib/libbluray
-
 ifeq (1,$(FEATURE.qsv))
 ifeq (,$(filter $(HOST.system),freebsd))
     MODULES += contrib/libvpl
